pkg/config/migration/js.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"bytes"
+	"fmt"
+
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/robertkrimen/otto"
+)
+
+const jsTransformContext = `
+// env2prop converts an environment variable to property name
```

Comment:
I'm sorry to be complicating things here, but what if you were to move these nested in-line JS blocks into their own files, and loaded them in? That way they could have proper unit tests executing them as a part of your test suite.

Reply:
That's a good idea, I've now added test-cases specifically for exercising these functions and their go equivalents.